Specs/Scene/ModelSpec.js

```diff
@@ -2735,28 +2737,61 @@ defineSuite([
 
     it('renders with the KHR_materials_pbrSpecularGlossiness extension', function() {
         return loadModel(boomBoxPbrSpecularGlossinessUrl).then(function(model) {
+            model.show = false;
+            model.zoomTo(2);
+
+            var originalColor;
+            expect(scene).toRenderAndCall(function(rgba) {
+                originalColor = rgba;
+            });
+
             model.show = true;
-            model.zoomTo();
 
-            scene.renderForSpecs();
+            expect(scene).toRenderAndCall(function(rgba) {
+                expect(rgba).not.toEqual(originalColor);
+            });
+
+            primitives.remove(model);
         });
     });
 
     it('renders with the KHR_materials_pbrSpecularGlossiness extension with defaults', function() {
         return loadModel(boomBoxPbrSpecularGlossinessDefaultsUrl).then(function(model) {
+            model.show = false;
+            model.zoomTo(2);
+
+            var originalColor;
+            expect(scene).toRenderAndCall(function(rgba) {
+                originalColor = rgba;
+            });
+
             model.show = true;
-            model.zoomTo();
 
-            scene.renderForSpecs();
+            expect(scene).toRenderAndCall(function(rgba) {
+                expect(rgba).not.toEqual(originalColor);
+            });
+
+            primitives.remove(model);
         });
     });
 
     it('renders with the KHR_materials_pbrSpecularGlossiness extension when no textures are found', function() {
         return loadModel(boomBoxPbrSpecularGlossinessNoTextureUrl).then(function(model) {
+            model.show = false;
+            model.zoomTo(2);
```

Comment:
Here and throughout, `2` should be `2.0`

https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md#basic-code-construction